src/ray/core_worker/context.cc

```diff
@@ -20,7 +20,9 @@ struct WorkerThreadContext {
     put_index = 0;
   }
 
-  void SetCurrentTask(const TaskSpecification &spec) { SetCurrentTask(spec.TaskId()); }
+  void SetCurrentTask(const TaskSpecification &task) {
```

Comment:
I would rename these arguments `task_spec` to make it clear that it is not a `Task` (which includes runtime information). Same elsewhere.